code/olc.c

```diff
@@ -1475,7 +1475,7 @@ void do_resets(CHAR_DATA *ch, char *argument)
 
 				if (get_mob_index(is_number(arg3) ? atoi(arg3) : 1) == nullptr)
 				{
-					send_to_char("Mob no existe.\n\r", ch);
+					send_to_char("Mob no exists.\n\r", ch);
```

Comment:
This was meant to be connected with the comment above
```suggestion
					send_to_char("Mob does not exist.\n\r", ch);
```